Apps/Playground/UWP/App.cpp

```diff
@@ -215,24 +215,36 @@ void App::OnPointerMoved(CoreWindow^, PointerEventArgs^ args)
     {
         const auto& point = args->CurrentPoint;
         const auto& position = point->RawPosition;
+        const auto& deviceType = args->CurrentPoint->PointerDevice->PointerDeviceType;
+        const auto& deviceSlot = args->CurrentPoint->PointerId;
 
-        m_nativeInput->MouseMove(static_cast<int>(position.X), static_cast<int>(position.Y));
-
-        /*
-         * Note: Because PointerPressed only first for the first button press (and no additional ones)
-         * and PointerReleased only fires when all buttons have been released, we need an alternative way
-         * to track additional button presses.
-         * (See Definition - PointerPressed: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerpressed?view=winrt-22000)
-         * (See Remarks - PointerReleased: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerreleased?view=winrt-22000)
-         *
-         * In both of the below functions, we have three booleans that track each button's state and then we
-         * run these two functions to update any buttons that we may have missed.
-         */
-
-        if (point->IsInContact)
+        if (deviceType == Windows::Devices::Input::PointerDeviceType::Mouse)
         {
-            OnPointerPressed(nullptr, args);
-            OnPointerReleased(nullptr, args);
+            m_nativeInput->MouseMove(static_cast<int>(position.X), static_cast<int>(position.Y));
+
+            /*
+             * Note: Because PointerPressed only first for the first button press (and no additional ones)
```

Comment:
I think this is supposed to say the following?
```suggestion
             * Note: Because PointerPressed only fires for the first button press (and no additional ones)
```

Reply:
Good catch!  I'll get that fixed up